### ISSUE TITLE
updates?

### ISSUE THREAD

**Issue by mcgriffiths** [jfjelstul/englishfootball#4]
Do you have any plans to update the data for seasons more recent than 2021/22?

**Comment by jfjelstul** [comment]
Yes, I'm going to add the 2022/23 and 2023/24 seasons after the current season ends in a few weeks.

**Comment by mcgriffiths** [comment]
That's great, thanks! I came across your database today, curious about how my team Huddersfield's failure to finish in the top two of any division since 1979/80 compares to others. Answer: very badly.

**Comment by rmontestt** [comment]
> That's great, thanks! I came across your database today, curious about how my team Huddersfield's failure to finish in the top two of any division since 1979/80 compares to others. Answer: very badly.

Updated https://github.com/rmontestt/englishfootball/tree/add-2024-25-and-2025-26-seasons
